cli/src/main/kotlin/norm/cli/NormCli.kt

```diff
@@ -77,6 +81,14 @@ class NormCli : CliktCommand( // command name is inferred as norm-cli
             IO(sqlFile, basePath, outDir).process(normApi::generate)
         }
     }
+
+    fun modifiedFilesFromGitOnly(): List<String> {
+        val builder = FileRepositoryBuilder()
+        val repo = builder.setGitDir(File("." + "/.git")).setMustExist(true)
```

Comment:
- [ ]  Get the diff only if .git repository exists otherwise fallback to full file list
- [ ]  Add tests